survlearners/R/surv_s_grf.R

```diff
@@ -1,20 +0,0 @@
-#' @title S-learner for survival data implemented via grf::survival_forest
-#'
-#' @description  S-learner via grf::survival_forest
-#'
-#' @param data training data
```

Comment:
remember Build->Document

Reply:
ah, good catch!

Reply:
This .Rd file update comes up whenever I delete/add functions/packages, any other situations you have encountered?

Reply:
Yes, any time you touch a `# ' @ ...` section in an R file